### PR TITLE
Refactor DataUpdateScript to make its code simpler

### PR DESCRIPTION
**Pull request by turara**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Remove unnecessary where method and Pathname initialization.

About where method, it was originally introduced to select ids of enqueued scripts, so it's no longer needed with enqueued method.

## Added tests?

- [ ] Yes
- [x] No, and this is why: because methods changed already have tests or are private.
- [ ] I need help with writing tests

## Added to documentation?

- [ ] Docs.forem.com
- [ ] README
- [x] No documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @turara, would you be willing to explain the PR? It's unclear to me what problem you're trying to tackle. Thank you!

**Comment by turara** [comment]
@rhymes Thank you for your comment!

I was trying to improve inefficient code and make them simpler. Is it not recommended to make a PR for that? If so, it's my bad.

File: app/models/data_update_script.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class << self
     def scripts_to_run
       insert_new_scripts
 
-      enqueued.where(id: ids).order(file_name: :asc)
```
Comment (mstruve) [inline app/models/data_update_script.rb:15]: Ummm, I have no idea why we did it this way but great catch!!!

File: app/models/data_update_script.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def scripts_to_run?
 
     def filenames
       Dir.glob("*.rb", base: DIRECTORY).map do |f|
-        Pathname.new(f).basename(".rb").to_s
+        File.basename(f, ".rb")
```
Comment (mstruve) [inline app/models/data_update_script.rb:32]: According to my testing, this looks to return the same thing but is a bit simpler to read.

**Review by mstruve — APPROVED** [review]
Welcome to the Forem project @turara! 

Not sure how we let that `ids` issue slip through the cracks but really nice catch on that one. Good work here! Thanks for the PR! 

One small bit of feedback I have is you might want to try making the title a little more descriptive next time so it gives a hint as to what the PR is trying to accomplish. `Update data_update_script.rb` is just a tad vague 😊

File: app/models/data_update_script.rb (hunk 1 above)
Comment (turara) [reply app/models/data_update_script.rb:15]: Git blame told me that `ids` was originally meant to be load_script_ids below :)

```ruby
    def load_script_ids
      filenames.
        map { |file_name| find_or_create_by(file_name: file_name) }.
        select(&:enqueued?).
        map(&:id)
    end
```

**Comment by turara** [comment]
@mstruve Thank you for your review and feedback! I changed the title a bit.

**Review by rhymes — APPROVED** [review]
Thank you @turara!

**Pull request merged**